dateparser/date.py

```diff
@@ -214,19 +216,23 @@ def _is_valid_date_obj(self, date_obj):
 
 class DateDataParser(object):
 
-    def __init__(self, language=None, allow_redetect_language=False):
-        if isinstance(language, basestring):
+    def __init__(self, languages=None, allow_redetect_language=False):
+        if isinstance(languages, list):
             available_language_map = default_language_loader.get_language_map()
-            if language in available_language_map:
-                language = available_language_map[language]
+
+            if all([language in available_language_map for language in languages]):
+                languages = [available_language_map[language] for language in languages]
             else:
-                raise ValueError("Unknown language %r" % language)
+                unsupported_languages = set(languages) - set(available_language_map.keys())
+                raise ValueError("Unknown language(s) %r" % ', '.join(unsupported_languages))
+        elif languages:
```

Comment:
So, if we check for valid input in first if clause, and here we want to check if anything else was passed (and we have None as a default valid value), then i would say `elif languages is not None` would be appropriate.